R/read_forresdat.R

```diff
@@ -2,13 +2,20 @@
 #'
 #' This function reads a table in git2rdata format from git repository forresdat
 #' (or another git repository dependent on the given repo path).
+#' Data available in forresdat only contains observations, so no records with
+#' zero values are added for for instance species that were not observed and
+#' hence absent.
+#' These zero value records can easily be added by using the function `add_zeros()`.
 #'
 #' @param tablename name of the table that should be read
 #' @param repo_path name and path of local git repository from which data should be retrieved
 #' @param join_plotinfo should table plotinfo be joined to the chosen table to
 #' add columns plottype and forest_reserve?  Default is TRUE.  (This is only
 #' possible if the given table contains a column plot_id, so this parameter
 #' should be put FALSE if this column is absent.)
+#' @param plottype Data of which 'plottype' (used method) should be retrieved?
+#' Default is 'CP' or 'circle plot', alternatively 'CA' or 'core area' could be
+#' chosen.
 #'
 #' @return A dataframe with the specified table, default columns plottype and forest_reserve.
```

Comment:
```suggestion
#' @return A dataframe with the specified table, and as a default extra columns plottype, forest_reserve and survey_number.
```
Dat is inderdaad wat ik als output verkies: de basis tabel, aangevuld met plottype, forest_reserve en survey_number, maar zonder de andere velden van plotinfo (zoals year, survey_reg/veg/dead/trees, data_processed)